CONTRIBUTION.md

```diff
@@ -0,0 +1,76 @@
+
+# How to contribute
+
+Pull requests should include the following:
+- link to an issue that describing the problem, or a description in the pull request describing the issue and what it solves
+- unit tests (if applicable)
+- storybook (if applicable)
+- a separate commit that contains the ouput from running `npm run docs`
+
+## Setup guide
+
+### Cloning repository
+
+Start by cloning the repository to desired directory.
+
+```
+git clone https://github.com/equinor/esv-intersection.git
+```
+
+### Install dependencies
+
+Install all dev dependencies defined in package.json using node.
+
+```
+npm install
+```
+
+## Usage
+
+### Storybook
+
+```
+npm run storybook
+```
+
+### Document generation
+
+```
+npm run docs
+```
+
+Three part process:
```

Comment:
This is a bit unclear. What is the exact relation between `npm run docs` above, and the three part process described here?

Reply:
the three part process describes what happens when running the script command. it's an over-explain and we can shorten it to something like: 'this command will overwrite the old documentation'